src/libwalletqt/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+set(PROJECT walletqt)
+
+file(GLOB_RECURSE SOURCE_FILES *.cpp)
+file(GLOB_RECURSE HEADER_FILES *.h)
+
+if(APPLE)
+    list(APPEND SOURCE_FILES "../qt/macoshelper.mm")
+endif()
+
+file(GLOB SOURCES_FOREIGN
```

Comment:
Ideally this will go to separate libraries, but we're not there yet.

Reply:
I have seen your comment here, but I think if we merge this as is this get even more complicated and we are just moving files around at some point.